ArduCopter/events.cpp

```diff
@@ -306,6 +306,77 @@ void Copter::gpsglitch_check()
     }
 }
 
+// executes lean-angle failsafe if lean angle is too high for longer than a set threshold
+// indicates the wind is too high to fly safely in planck mode
+void Copter::failsafe_lean_check(uint16_t duration_ms)
+{
+    // trigger with X seconds of failures while in a PLANCK mode
+    bool valid_mode = (control_mode == Mode::Number::PLANCKTRACK ||
```

Comment:
The idea is it should only trigger a failsafe if it's actively tag tracking, since the intent is to warn/act if it can't keep the tag in frame during a steady hover. If it's in, say, loiter or stabilize, the pilot is going to be correcting for the wind, and we shouldn't interfere. Similar thinking for WINGMAN, the pilot is providing the corrective action, and it's less important to keep the tag in frame